client/components/room/Video.jsx

```diff
@@ -289,37 +280,30 @@ class Video extends React.Component {
     pc = null;
   }
 
-  // handleVideoScreen() {
-  //   this.setState({
-  //     constraints: {
-  //       video: !this.state.constraints.video,
-  //       audio: this.state.constraints.audio
-  //     },
-  //     video: !this.state.constraints.video ? 'Video Off' : 'Video On'
-  //   }, () => (this.start()));
-  // }
-
-  // handleAudio() {
-  //   this.setState({
-  //     constraints: {
-  //       video: this.state.constraints.video,
-  //       audio: !this.state.constraints.audio
-  //     },
-  //     mute: !this.state.constraints.audio ? 'Mute' : 'Unmute'
-  //   }, () => (this.start()));
-  //   navigator.mediaDevices.getUserMedia(this.state.constraints)
-  //     .then(this.successCallback)
-  //     .catch(this.errorCallback);
-  // }
+  toggleVideo() {
+    if(localStream.getVideoTracks()[0].enabled) {
+      localStream.getVideoTracks()[0].enabled = false;
+    } else {
+      localStream.getVideoTracks()[0].enabled = true;
+    }
+  }
+
+  toggleAudio() {
```

Comment:
We might be able to make this more readable by saving it to a convenience variable, "audioEnabled"  then `audioEnabled ? !audioEnabled : audioEnabled` Not sure though.  Just a thought